gui-wm/hyprland/hyprland-0.42.0-r1.ebuild

```diff
@@ -46,25 +57,22 @@ RDEPEND="
 	x11-libs/libxkbcommon
 	x11-libs/pango
 	x11-libs/pixman
-	X? (
-		x11-libs/libxcb:0=
-	)
+	${XWAYLAND_DEPEND}
 "
 DEPEND="
 	${RDEPEND}
-	${WLROOTS_DEPEND}
 	>=dev-libs/hyprland-protocols-0.3
 	>=dev-libs/hyprlang-0.3.2
 	>=dev-libs/wayland-protocols-1.36
 	>=gui-libs/hyprutils-0.2.1
 "
 BDEPEND="
-	${WLROOTS_BDEPEND}
 	|| ( >=sys-devel/gcc-13:* >=sys-devel/clang-16:* )
 	app-misc/jq
 	dev-build/cmake
 	>=dev-util/hyprwayland-scanner-0.3.8
 	virtual/pkgconfig
+	${XWAYLAND_DEPEND}
```

Comment:
I think ${XWAYLAND_DEPEND} should be removed from BDEPEND.

BDEPEND specifies dependencies applicable to CBUILD, usually BDEPEND won't contribute to link/run time phase.

https://devmanual.gentoo.org/general-concepts/dependencies/index.html#build-dependencies